nautobot/dcim/templates/dcim/module_list.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'generic/object_list.html' %}
+
+{% block bulk_buttons %}
+    {% if perms.dcim.change_device %}
```

Comment:
Same comment here on permissions -- though in this case, if we do keep it, shouldn't it be `change_module` rather than `change_device`?